asserts/device_asserts.go

```diff
@@ -25,6 +25,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/snapcore/snapd/snap"
```

Comment:
by policy we can't import snap from asserts, asserts is used as a library and snap brings in too much

Reply:
Bummer. I suppose we'll have to copy it over and keep both in sync. OTOH, there's not much change to be expected in the area of snap name validation.

Reply:
the other approach is to create a package snap/validate with all the Validate* helpers but not the main snap one